src/com/rcv/Tabulator.java

```diff
@@ -456,16 +459,16 @@ private boolean hasContinuingCandidates(
       }
     }
 
-    // loop over the ballots and count first-place votes, considering only continuing candidates
-    for (int i = 0; i < allSortedRankings.size(); i++) {
+    // loop over the ballots and count votes for continuing candidates
+    for (int i = 0; i < castVoteRecords.size(); i++) {
       if (exhaustedBallots.get(i) != null) {
```

Comment:
Now that we're storing tabulation state in the CVR, we should maybe move this in there also. (No need to do that in this diff.)